airbyte-integrations/connectors/source-surveymonkey/unit_tests/test_streams.py

```diff
@@ -225,125 +225,6 @@ def test_surveys(requests_mock):
     ]
 
 
-def test_survey_pages(requests_mock):
```

Comment:
Why did we delete those tests? They seems like they should continue to work if we extract the proper stream from the source, something like:

```
    stream = next(filter(lambda x: x.name == "survey_pages", SourceSurveymonkey().streams(<...>)))
    records = list(map(lambda record: record.data, stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice={"survey_id": "307785415"})))
```

Reply:
The survey_pages stream has been migrated to low code, and all functionality has already been tested with low-code CDK unit tests.

I deleted this because we're trying to get rid of unnecessary tension in tests.

Reply:
Low-code CDK unit tests does not ensure that the implementation for source-surveymonkey respect the Survey Monkey API. Those tests are useful. If we had those, we would have caught https://github.com/airbytehq/airbyte/pull/35561#discussion_r1513129701 for example

Reply:
Unit tests added

Reply:
I still see those tests as deleted [here](https://github.com/airbytehq/airbyte/pull/35561/files/e77c6d825abe5d338c5db162dd54777935860190#diff-c13aa76ab6ef61dabc9d883effa23d94f6e161790b77873f83d4eb1fbc8c9bebL228). Where are those tests?

Reply:
I was do some refactoring with parametric decorator and those test you can find in unit_tests/test_streams.py line 62 (test_survey_data)